### PR TITLE
version 0.9.0 improvements

### PR DESCRIPTION
**Pull request by fmigneault**
* add permissions config to auto-generate user/group rules on startup
* fix many invalid or erroneous swagger specifications
* attempt db creation on first migration if not existing
* add continuous integration testing and deployment (#139)
* reduce excessive sqlalchemy logging using `MAGPIE_LOG_LEVEL >= INFO`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,91 +101,183 @@ clean-test:
...
 	@echo "Starting Cron service..."
 	cron
 
 .PHONY: start
-start: cron install
+start: install
 	@echo "Starting Magpie..."
-	exec gunicorn -b 0.0.0.0:2001 --paste "$(CUR_DIR)/magpie/magpie.ini" --workers 10 --preload
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		exec gunicorn -b 0.0.0.0:2001 --paste "$(CUR_DIR)/magpie/magpie.ini" --workers 10 --preload &'
```
Comment (dbyrns) [inline Makefile]: That little & at the end of the command looks like a hack, no?

File: magpie/register.py [hunk 2]
```diff
@@ -25,6 +49,11 @@
 SERVICES_PHOENIX = 'PHOENIX'
```
Comment (dbyrns) [inline magpie/register.py]: Is this file only related to Phoenix? Because all Phoenix related stuff could be marked as deprecated... and removed as soon as the latest twitcher version is used.

File: Makefile (hunk 1 above)
Comment (fmigneault) [reply Makefile]: Tells bash to not wait for the return of the call.
So basically, run the process detached.

File: magpie/register.py (hunk 2 above)
Comment (fmigneault) [reply magpie/register.py]: Indeed, all of Phoenix related code is preserved for back compatibility to old twitcher, but is mostly deprecated now.
